### PR TITLE
Clarify what `Cell::replace` returns

### PR DESCRIPTION
**Pull request by camelid**

### CODE REVIEW

File: library/core/src/cell.rs [hunk 1]
```diff
@@ -374,7 +374,7 @@ impl<T> Cell<T> {
         }
     }
 
-    /// Replaces the contained value, and returns it.
+    /// Replaces the contained value with `val`, and returns the old contained value.
```
Comment (pickfire) [inline library/core/src/cell.rs:377]:
```suggestion
    /// Replaces the wrapped value with a new one, returning the old value,
    /// without deinitializing either one.
    ///
    /// This function corresponds to [`std::mem::replace`](../mem/fn.replace.html).
```

Why not just use the same one as `RefCell::replace`?

**Comment by JohnCSimon** [comment]
Ping from triage
@camelid - can you please address the comment from the reviewer? Thank you.

@rustbot label: -S-waiting-on-review +S-waiting-on-author

**Comment by camelid** [comment]
> can you please address the comment from the reviewer? Thank you.

I was waiting for a review from @cramertj, the assignee.

**Comment by Mark-Simulacrum** [comment]
@bors r+ rollup

I think this is good. I'm not convinced that reference to mem::replace or the alternative wording is better.

**Pull request merged**